playbook.yaml

```diff
@@ -0,0 +1,8 @@
+- hosts: localhost
+  connection: local
+  gather_facts: false
+  vars_files:
+    - roles/cluster_template/vars/vars.yaml
+    - roles/cluster_template/vars/constants.yaml
```

Comment:
You should not be including vars files from inside your roles. If you configure `roles/<role>/vars/main.yaml` or `roles/<role>/defaults/main.yaml` they will be discovered automatically by the role. In general prefer `defaults/main.yaml` over `vars/main.yaml` because the values in `defaults` are designed to be [easy to override](https://docs.ansible.com/ansible/2.9/user_guide/playbooks_reuse_roles.html#role-default-variables).

If you need vars files from outside the role, put them somewhere else.

Ansible will load `group_vars/all.yaml` automatically (or `group_vars/all/<whatever>.yaml` if you would like more than one file).